File: compose/src/main/kotlin/org/dravensoft/arena/tokens/ArenaContrast.kt
package org.dravensoft.arena.tokens

import androidx.compose.ui.unit.Dp

public object ArenaContrast {
    public fun border(increased: Boolean): Dp = if (increased) ArenaTokens.bwStrong else ArenaTokens.bw

    public fun focusWidth(increased: Boolean): Dp = if (increased) ArenaTokens.sp1 else ArenaTokens.focusWidth

    public fun scrimBlur(reduced: Boolean): Dp = if (reduced) ArenaTokens.sp0 else ArenaTokens.scrimBlur
}

File: swiftui/Sources/ArenaTokens/tokens/ArenaContrast.swift
import CoreGraphics

public enum ArenaContrast {
    public static func border(_ increased: Bool) -> CGFloat { increased ? ArenaTokens.bwStrong : ArenaTokens.bw }

    public static func focusWidth(_ increased: Bool) -> CGFloat { increased ? ArenaTokens.sp1 : ArenaTokens.focusWidth }

    public static func scrimBlur(_ reduced: Bool) -> CGFloat { reduced ? ArenaTokens.sp0 : ArenaTokens.scrimBlur }
}
